src/scenes/PlantsDashboardRouter/components/HighestAndLowestMortalityRateZonesCard.tsx

```diff
@@ -16,20 +17,35 @@ export default function TotalMortalityRateCard(): JSX.Element {
   const [highestZoneId, setHighestZoneId] = useState<number>();
   const [lowestZoneId, setLowestZoneId] = useState<number>();
 
+  const isSurvivalRateCalculationEnabled = isEnabled('Survival Rate Calculation');
+
   useEffect(() => {
     let _highestMortalityRate = 0;
     let _lowestMortalityRate = 100;
     let _highestZoneId: number | undefined;
     let _lowestZoneId: number | undefined;
     observationSummaries?.[0]?.plantingZones.forEach((zone: PlantingZoneObservationSummary) => {
-      if (zone.mortalityRate !== undefined) {
-        if (zone.mortalityRate >= _highestMortalityRate) {
-          _highestMortalityRate = zone.mortalityRate;
-          _highestZoneId = zone.plantingZoneId;
+      if (isSurvivalRateCalculationEnabled) {
+        if (zone.survivalRate !== undefined) {
+          if (zone.survivalRate >= _highestMortalityRate) {
+            _highestMortalityRate = zone.survivalRate;
+            _highestZoneId = zone.plantingZoneId;
```

Comment:
We'll probably want to rename these variables at some point before we remove the feature flag, but can be done in a later PR

Reply:
Yes, I will do it when removing the feature flag!